bodhi/tests/server/views/test_generic.py

```diff
@@ -53,6 +53,16 @@ def test_home(self):
         self.assertIn('Log out', res)
         self.assertIn('Fedora Update System', res)
 
+    @unittest.skipIf(six.PY3, 'Not working with Python 3 yet')
+    def test_critical_update_link_home(self):
+        update = Update.query.first()
+        update.critpath = True
+        update.status = UpdateStatus.testing
+        self.db.commit()
+        res = self.app.get('/', headers={'Accept': 'text/html'})
```

Comment:
I recommend putting empty lines before and after this line. It's not a required style so you can keep it this way if you prefer, but I find it to be nice to break tests into three code blocks, most of the time:

0. Test set up.
1. Execute the code you are testing.
2. Assert that the right thing happened.

By adding spaces above and below this line, it makes it easier to spot the three segments of the test, and highlights that we are testing the call to ```/```.